app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self.health <= 0:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (f"{{"
+                f"Name: {self.name},"
+                f" Health: {self.health},"
+                f" Hidden: {self.hidden}"
+                f"}}"
+                )
```

Comment:
Your opening parenthesis has arguments on the same line, position your closing parenthesis at the end of the previous line after string to match the opening one.